tower-http/src/validate_request.rs

```diff
@@ -112,6 +178,8 @@
 //! # Ok(())
 //! # }
 //! ```
+//!
+//! [`Accept`]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept
```

Comment:
not too sure about this one... 

I think that it is mostly useless because I didn't find any link to this in the doc... but I copy/paste it from the existing `accept()` function documentation 🤷